### PR TITLE
Update master Roadmap

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Updates the roadmap in the master branch, updating the release list and the MRTK vNext plan and branch/release strategy.

Also increments the version file, which was slightly lagging behind the actual release.

Changes
---
- Related to #2659.

### CODE REVIEW

File: Roadmap.md [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
+
+For the [master](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master) branch releases, development is being done in [htk_development](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/htk_development). Please refer to the [Upcoming releases table](#upcoming-releases), or the [Projects](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects) page for specifics.
 
 Please see [Branch management plan](#branch-management-plan) for more details.
 
+## MRTK Version Next
 
-# MRTK Version Next
-MixedRealityToolkit Version Next aims to further extend the capabilities of the toolkit and also introduce new features, including the capability to support more VR/AR/XR platforms beyond Microsoft's own Windows Mixed Reality platform.
+MixedRealityToolkit Version Next aims to further extend the capabilities of the toolkit and also introduce new features, including the capability to support more VR / AR / XR platforms beyond Microsoft's own Windows Mixed Reality platform.
```
Comment (StephenHodgson) [inline Roadmap.md]: This might be a silly request but I think the order of the realities is important:

` AR / XR / VR ` to reflect the spectrum better.

Comment (Ecnassianer) [reply Roadmap.md]: Do you see XR as a segment of the spectrum, rather than the entire spectrum itself?

Comment (StephenHodgson) [reply Roadmap.md]: Usually I refer to the whole spectrum as Mixed Reality not XR.

Comment (Ecnassianer) [reply Roadmap.md]: I've always read XR as "<Wildcard> Reality", thus the whole spectrum. But you know, we're all making this stuff up as we go, that's the fun part. :)

Comment (keveleigh) [reply Roadmap.md]: I also tend to use MR instead of XR.

I updated the order of this list.

Comment (Ecnassianer) [reply Roadmap.md]: I only see spaces your commit. :D

**Pull request merged**